shuffle/README.md

```diff
@@ -103,3 +103,21 @@ newly created account, (3) publish another `SampleModule::Coin` resource to the
 
 The first two transactions should succeed and the last one should fail because you can only publish
 one resource of each type under an address.
+
+## Step 4: Publish Modules
+
+Write any module, like SampleUserModule.move in `/move/src/modules/user_modules` and put in in the user_modules directory.
+
+Inside `./sample-app`, run the following command:
```

Comment:
If we've removed the `sample-app`, let's remove this part of the README about the sample app.

Reply:
I think we can remove this README entirely actually and replace it with yours